### PR TITLE
Added snapstack template.

### PR DESCRIPTION
**Pull request by pengale**
Allows us to test the snap against a basic Openstack deploy.

@coreycb

### CODE REVIEW

File: tests/{{cookiecutter.repo_name}}.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -ex
+
+source $BASE_DIR/admin-openrc
+
```
Comment (coreycb) [inline tests/{{cookiecutter.repo_name}}.sh:6]: Maybe worth adding a comment at the end of this file to say something like "here's where you put your setup and tests"

File: tests/snapstack_test.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+class SnapstackTest(unittest.TestCase):
+
+    def test_snapstack(self):
+        '''
+        _test_snapstack_
+
+        Run a basic smoke test, utilizing our snapstack testing harness.
+
+        '''
```
Comment (coreycb) [inline tests/snapstack_test.py]: A comment here might be worthwhile along the lines mentioned in the keystone snap review, to mention what other default  base snaps are deployed alongside this snap and it's tests.

**Review by coreycb — COMMENTED** [review]
Just a couple of comments on comments.

File: tests/{{cookiecutter.repo_name}}.sh (hunk 1 above)
Comment (pengale) [reply tests/{{cookiecutter.repo_name}}.sh:6]: Done.

**Comment by coreycb** [comment]
LGTM, thanks!

**Pull request merged**